src/main/java/uk/ac/ebi/eva/submission/model/SubmissionStatus.java

```diff
@@ -0,0 +1,78 @@
+package uk.ac.ebi.eva.submission.model;
+
+import org.springframework.lang.NonNull;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+@Entity
+@Table(name = "submission_status")
+public class SubmissionStatus {
+
+    public SubmissionStatus() {
+
+    }
+
+    public SubmissionStatus(String submissionId) {
+        this.submissionId = submissionId;
+    }
+
+    @Id
+    @NonNull
+    @Column(nullable = false, name = "submission_id")
+    private String submissionId;
+
+    @NonNull
+    @Column(nullable = false, columnDefinition = "BOOLEAN DEFAULT FALSE")
+    private boolean completed;
+
+    @Column(nullable = false, columnDefinition = "TIMESTAMP DEFAULT CURRENT_TIMESTAMP", insertable = false, updatable = false)
+    private LocalDateTime dirCreatedTime;
```

Comment:
I think the date you have here should be the submission initiation date
We will have to keep more date than this one though. At least the the start and end for submission will be required

Reply:
added initiation-date, uploaded-date and completion-date